### PR TITLE
Update the site with some more info

### PR DESCRIPTION
**Pull request by bcomnes**
Not ready to merge yet, but I want to make the site have a little more info on it.  Feel free to check out he site preview.

### CODE REVIEW

**Comment by bcomnes** [comment]
Ok this is ready to go whenever.

**Review by ungoldman — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,45 @@
-# hypermodules
+<h1 align="center">✨<a href="https://github.com/hypermodules">hypermodules</a>⚡️</h1>
 
-- [github.com/hypermodules](https://github.com/hypermodules)
-- [github.com/hypermodules/hyperamp](https://github.com/hypermodules/hyperamp)
-- [github.com/hypermodules/dti](https://github.com/hypermodules/dti)
-- [github.com/hypermodules/entypo](https://github.com/hypermodules/entypo)
```
Comment (ungoldman) [inline README.md:6]: why remove dti and entypo?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,45 @@
-# hypermodules
+<h1 align="center">✨<a href="https://github.com/hypermodules">hypermodules</a>⚡️</h1>
```
Comment (ungoldman) [inline README.md]: i think it looks better with sparkles on both sides

### **✨ hypermodules ✨**

more magical

Comment (bcomnes) [reply README.md]: ok ill change it.  It looks pretty scrappy either way.  I might try to make it look more professional at some point.

File: README.md (hunk 1 above)
Comment (bcomnes) [reply README.md:6]: Didn't have time to make screenshots, but I'm open to adding them back when someone has time to with screenshots.

**Pull request merged**